specs/DesignDiscussions/InputLabels.md

```diff
@@ -72,17 +72,55 @@ This conflict speaks to a general tension inherent in the design of Adaptive Car
 
 In order to gather more data to help us balance the needs of the card authors and hosts, we will send a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
 
+> ## Rendering for string labels
+>
+> The default rendering for labels should be performed with the default values for all of the following properties:
+> * spacing
+> * fontFamily
+> * fontSizes
+> * fontWeights
+> * foregroundColors
+>
+> This would allow authors to emulate writing a simple TextBlock element which only non-default property would be the text, making the string version look like this
```

Comment:
>which only non-default property would be the text, [](start = 73, length = 50)

I am not sure i can parse this correctly? Do you mean the only non-default property would be the text that is specified as value for the newly-introduced Label property?